codalab/worker/docker_image_manager.py

```diff
@@ -218,6 +219,15 @@ def download():
                             'message'
                         ] = "Can't download image: {}".format(ex)
 
+                # Check docker image size before pulling from docker hub.
+                # Images with size larger than '10g' will be rejected to download.
+                size_validated, failure_msg = docker_utils.check_image_size_before_pulling(
```

Comment:
The logic here can just be get the size, and just check whether we're exceeding the limit.

Reply:
Updated.